infrastructure-playbooks/rolling_update.yml

```diff
@@ -341,6 +341,7 @@
     health_osd_check_retries: 40
     health_osd_check_delay: 30
     upgrade_ceph_packages: True
+    ceph_release: "{{ ceph_stable_release }}"
```

Comment:
isn't `ceph_release` already set in ceph-common/ceph-container-common ? why do we have to set it here ?